### PR TITLE
Deprecated instanciate in favor of instantiation (correct grammatical…

### LINKED ISSUE #97: Instanciate isn't the correct form
Instanciate is the french form of the verb, instantiate being the english form of the same verb.
Same goes for instanciation that must be changed to instantation.
[Source](https://www.linguee.fr/anglais-francais/traduction/instantiate.html),
it's also worth mentioning that no result comes up while searching for "instanciate" in a web navigator

### PR DESCRIPTION
**Pull request by Eliott-Guevel**
Deprecated instanciate in favor of instantiation (correct grammatical form). Same for instanciation which needs to be replaced by instantiation.
This commit fixes #97

### CODE REVIEW

**Review by labordep — CHANGES_REQUESTED** [review]
You need to use auto correction function of the deprecation :

```smalltalk
self deprecated: 'Example of a deprecated method with transform'
       transformWith: '`@receiver deprecatedMethod' -> '`@receiver newMethod'
```

**Comment by labordep** [comment]
Waiting for https://github.com/pharo-project/pharo/issues/14697 analysis.

**Review by labordep — CHANGES_REQUESTED** [review]
Deprecated methods should call directly the new methods. This is not useful to conserve each old implementation.
For example : 

```smalltalk
MolComponentImpl classSide >> instanciate 

self deprecated: 'Use instantiate' transformWith: '`@receiver instanciate' -> '`@receiver instantiate'.

^ self instantiate 

```

**Comment by labordep** [comment]
> Deprecated methods should call directly the new methods. This is not useful to conserve each old implementation.
> 
> For example : 
> 
> 
> 
> ```smalltalk
> 
> MolComponentImpl classSide >> instanciate 
> 
> 
> 
> self deprecated: 'Use instantiate' transformWith: '`@receiver instanciate' -> '`@receiver instantiate'.
> 
> 
> 
> ^ self instantiate 
> 
> 
> 
> ```

@Eliott-Guevel have you seen my review ?

**Comment by Eliott-Guevel** [comment]
I've just made the adapted modifications

File: Molecule-Examples/MolGNSSGalileo.class.st [hunk 1]
```diff
@@ -0,0 +1,81 @@
+Class {
```
Comment (labordep) [inline Molecule-Examples/MolGNSSGalileo.class.st:1]: Examples from GPS should not be here (this is not a PR related to that)

File: Molecule-Tests/MolHomeServicesTest.class.st [hunk 2]
```diff
@@ -13,7 +13,7 @@ I do not worry about interactions between components. See:
 Class {
```
Comment (labordep) [inline Molecule-Tests/MolHomeServicesTest.class.st:13]: Can you reformat the code as previous ? (One line at each instructions)

**Review by labordep — CHANGES_REQUESTED** [review]
Examples from GPS should not be here (this is not a PR related to that) - see review for others feedback.
Ok for my first review.

**Review by Eliott-Guevel — COMMENTED** [review]
Removed classes enhancing the GPS example (are put in another branch of my Molecule fork which is associated to another pull request

**Comment by labordep** [comment]
Hum this PR is very strange because its appears that all the code of molecule have been modified !

**Comment by labordep** [comment]
I have the explanation :  

```
I just wanted to let people who are using Pharo12 that they might notice some changes while exporting their code to github now.

We are introducing a TonelV3 format. The goals are to:
- Follow the spec that was agreed upon because currently we have symbols instead of strings in some places and that made it incompatible with Gemstone or Vast 
- Add missing informations. This is because in Pharo we have packages and tags but the only thing exported is the category. So reading the categories was ambiguous and relied on some heuristic. Loading the same class in two pharo image could load the class in different packages depending on the state of the image. 
- We want to add meta data for new features (not yet done). For example there is a new feature to rename a class and keep the old name as a deprecated alias for a while. We would like to be able to declare those info in the fluid class definition and in the Tonel format.

The new format still exports the category (but there is a setting to not export it) to stay backward compatible for now. 
Using the Tonel paser to read classes that were exported before Pharo 12 can have some difficulties to manage the tags, but the TonelReader class that is used by Iceberg knows how to resolve all of those problem to ensure the classes are in the right packages and in the right classes!
```

So don't use Pharo 12 for this work. Can you re send your modifications with P11 ? Thanks.

**Comment by Eliott-Guevel** [comment]
I've made the modifications with Pharo 11, it should now only modify the relevant files

**Pull request closed without merging**